src/components/tables/examples/TableHeaderExamples.tsx

```diff
@@ -12,9 +14,7 @@ export class TableHeaderExamples extends React.Component<any, any> {
       {
         title: 'Second col'
       },
-      {
-        title: 'Third col'
-      },
+      new SortExample(SortTypes.None, () => { }),
```

Comment:
I'd leave the example here just so we can easily compare a column with sorting from the other ones.